pkg/devfile/parser/data/2.0.0/components.go

```diff
@@ -227,3 +228,81 @@ func (d *Devfile200) UpdateEvents(postStart, postStop, preStart, preStop []strin
 		d.Events.PreStop = preStop
 	}
 }
+
+// AddVolume adds the volume to the devFile and mounts it to all the container components
+func (d *Devfile200) AddVolume(volume common.Volume, path string) error {
+	for _, component := range d.Components {
+		if component.Container != nil {
+			for _, volumeMount := range component.Container.VolumeMounts {
+				if volumeMount.Path == path {
+					return fmt.Errorf("another volume, %s, is mounted to the same path: %s, on the container: %s", volumeMount.Name, path, component.Container.Name)
+				}
+			}
+			component.Container.VolumeMounts = append(component.Container.VolumeMounts, common.VolumeMount{
+				Name: volume.Name,
+				Path: path,
+			})
+		} else if component.Volume != nil && component.Volume.Name == volume.Name {
+			return fmt.Errorf("volume %s already exists", volume.Name)
```

Comment:
update to https://github.com/openshift/odo/blob/master/pkg/devfile/parser/data/common/errors.go#L13

Reply:
Fixed.